src/components/modal/modal.scss

```diff
@@ -5,6 +5,8 @@
  *
  * @prop --calcite-modal-content-background: Specifies the background color of content placed in the `content` slot.
  * @prop --calcite-modal-content-padding: Specifies the padding of the modal `content` slot.
+ * @prop --calcite-modal-content-header-padding: Specifies the padding of the modal `content-header` slot.
```

Comment:
cc @ashetland / @SkyeSeitz - slot names could be `content-top` and `content-bottom` to avoid confusion with existing elements.

Reply:
Would `content-start` and `content-end` be more inline with existing conventions or do we reserve start/end for slots on the x-axis?

Reply:
What do you mean by this?

Reply:
We use start/end in multiple places, for example icon-start/end and actions-end (I think we have actions-start somewhere?). So instead of naming these slots `content-top` and `content-bottom` would `content-start` and `content-end` make sense? Or would top/bottom be preferred here?

Reply:
I don't think it would hurt to reserve start & end with the x-axis in case we ever needed both top/bottom & start/end, so `content-top` & `content-bottom` would be my vote unless there were additional concerns.